### PR TITLE
Development branch

### PR DESCRIPTION
**Pull request by mosesmuchemi**
Requesting a code review for the embedding images and video project.

### CODE REVIEW

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="icon" type="image/x-icon" href="https://s.ytimg.com/yts/img/favicon-vfl8qSV2F.ico" />
+    <title>Youtube Clone</title>
+    <link rel="stylesheet" href="youtubestyle.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="navbar">
```
Comment (Torres-ssf) [inline index.html]: [OPTIONAL]
Using the `nav` tag for the navbar make your HTML code more semantic. This is a very important topic. You can learn more about [here](http://seekbrevity.com/semantic-markup-important-web-design/).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            <button type="submit"><i class="fa fa-search"></i></button>
+        
+            <div class="leftnav">
+                    <a href="#" title="Create a video or post"><i class="fa fa-video-camera"></i></a>
+                    <a href="#" title="Youtube apps"><i class="fa fa-th"></i></a>
+                    <a href="#" title="Messages"><i class="fa  fa-envelope"></i></a>
+            </div>
+        </div>
+    </div>
+    <div class="videobox">
```
Comment (Torres-ssf) [inline index.html:30]: [OPTIONAL]
It's also a good practice to use the `main` tag for the most relevant content of your page.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Great work team :clap: 
Nice job on this project. But before merging to master we need to make some changes to make your project even better.

[REQUIRED]
- [x] The search bar must be aligned to the middle of the navbar

[REQUIRED]
- [x] The video description section must have a 'share' button.

[REQUIRED]
- [x] The video description section must show the avatar of the user who uploaded the video

[REQUIRED]
- [x]  There are some errors in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.

[REQUIRED]
- [x] Since this project is an assignment is a good practice to include in the readme file the assignment link and also the GitHub account from the contributors. If you need a guide you can check this [one](https://github.com/Torres-ssf/apple-site-clone/blob/master/README.md)

[FRIENDLY-REMAINDER]
As it's said on the pathwrite, you should use GitHubFlow: For HTML & CSS projects. So for the next project try to give a feature name for the branch. :smile: 

Please make these changes and submit for code review again :+1: 
Happy conding :computer:

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Hi team, good job so far, we have made good progress. 
There seems to be an issue with the page layout though, the upnext sections show below the video and not to the side as it should

The readme file is not propely written please see recommendation from the previous reviewer, did you read that :thinking:

File: index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        </a>
+        <div class="search-container" style="padding-left: 350px">
```
Comment (Oghenebrume50) [inline index.html]: Inline styles are generally frowned upon, please move all inline styles to the css file

File: youtubestyle.css [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  float: left;
+  display: block;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+}
+
+.navbar input[type=text] {
+  /*float: right;*/
```
Comment (Oghenebrume50) [inline youtubestyle.css:27]: Please delete all commented codes

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Fix these things and submit for re-review :pray:

**Comment by Ispirett** [comment]
# Reviewing

File: index.html [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+            <iframe width="650" height="350" src="https://www.youtube.com/embed/V74l_zS1x8E"> </iframe>
+            <div class="videoInfo">
+                <p>
+                        Contributing to Open Source Part I: The Easy Way
+                    <span id="shareButton"><a href="#" title="share"><i class="fa fa-share"></i></a></span>
+                    <br>
+                </p>
+                <span><a href="#" title="Create a video or post"><i class="fa fa-user" aria-hidden="true"></i></a></span>
+                <span> The Odin Project</span>
+                <span id="shareButton">50,446 views</span>
```
Comment (Ispirett) [inline index.html:40]: You have duplicated  **ids**  please use a class instead

**Review by Ispirett — APPROVED** [review]
Hey!!!!!!!!!! Folks great job implementing changes :1st_place_medal: :100:  please make changes below and then you can complete this project.

#### Git/Githubflow reminder.

Friendly reminder: You should use Github/Git Flow in this project. However, changing it now would take too much time.
Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.

**Pull request merged**